lib/modules/datasource/conan/index.ts

```diff
@@ -62,10 +86,12 @@ export class ConanDatasource extends Datasource {
     packageName,
   }: GetReleasesConfig): Promise<ReleaseResult | null> {
     const depName = packageName.split('/')[0];
-    const userAndChannel = '@' + packageName.split('@')[1];
+    const userAndChannel = '@' + packageName.split('@')[1].split('#')[0];
+    const revision = getRevision(packageName);
     if (
       is.string(registryUrl) &&
-      ensureTrailingSlash(registryUrl) === defaultRegistryUrl
+      ensureTrailingSlash(registryUrl) === defaultRegistryUrl &&
+      is.undefined(revision)
```

Comment:
Does this new logic mean that we can only use getConanCenterReleases if there's no revisions?

Reply:
@segretil you need to implement `getDigest` function instead of returning the revision in `getReleases`

https://github.com/renovatebot/renovate/blob/2dbc15e6793916f75579d0a66126982d76304a3d/lib/modules/datasource/datasource.ts#L34


https://github.com/renovatebot/renovate/blob/2dbc15e6793916f75579d0a66126982d76304a3d/lib/modules/datasource/docker/index.ts#L816-L842


That will allow renovate to fetch the new `digest` to flow as `newDigest` to the auto replace function.

Reply:
@rarkins yes, I did not find a way to get the revision without using the Conan API.

Reply:
However it seems like the Conan API is working fast again https://github.com/conan-io/conan/issues/10298 so it might not be a big issue.